views/replication-controller.html

```diff
@@ -8,8 +8,9 @@
     <dt>Created</dt>
     <dd>{{resource.metadata.creationTimestamp | date:'medium'}}</dd>
     <dt>Replicas</dt>
-    <!-- TODO: Update for API version v1. The default has changed from 0 to 1. -->
-    <dd>{{resource.spec.replicas || 0}}</dd>
+    <dd>{{(resource.spec.replicas === undefined) ? 1 : resource.spec.replicas}}</dd>
+    <dt ng-if="resource.spec.serviceAccountName">Service account</dt>
+    <dd ng-if="resource.spec.serviceAccountName">{{resource.spec.serviceAccountName}}</dd>
```

Comment:
serviceAccountName is part of the pod spec inside the rc spec (rc.spec.template.spec.serviceAccountName)